homeassistant/components/screenlogic/sensor.py

```diff
@@ -90,3 +148,36 @@ def __init__(self, coordinator, pump, key):
     def sensor(self):
         """Shortcut to access the pump sensor data."""
         return self.coordinator.data[SL_DATA.KEY_PUMPS][self._pump_id][self._key]
+
+
+class ScreenLogicChemistrySensor(ScreenLogicSensor):
+    """Representation of a ScreenLogic IntelliChem sensor entity."""
+
+    def __init__(self, coordinator, key):
+        """Initialize of the pump sensor."""
+        super().__init__(coordinator, f"chem_{key}")
+        self._key = key
+
+    @property
+    def state(self):
+        """State of the sensor."""
+        value = self.sensor["value"]
+        return (
+            CHEM_DOSING_STATE.NAME_FOR_NUM[value]
+            if "dosing_state" in self._key
+            else value
+        )
```

Comment:
```suggestion
        if "dosing_state" in self._key:
            return CHEM_DOSING_STATE.NAME_FOR_NUM[value]
        return value
```

We prefer to unpack these if the ternary spans multiple lines

Reply:
Will do.

Reply:
https://github.com/home-assistant/core/pull/49689/commits/108c703acc2f390e57c6ec8ffdbcaabf56ac7dc3